src/finch/types/hris/employment_retrieve_many_params.py

```diff
@@ -18,6 +18,6 @@ class Request(TypedDict, total=False):
     """A stable Finch `id` (UUID v4) for an individual in the company.
 
     There is no limit to the number of `individual_id` to send per request. It is
-    preferantial to send all ids in a single request for Finch to optimize provider
+    preferential to send all ids in a single request for Finch to optimize provider
```

Comment:
note: this sentence comes from your OpenAPI spec so the easiest fix would be to update your OpenAPI spec directly.